backend/sirius-web-diagrams/src/main/java/org/eclipse/sirius/web/diagrams/components/NodeComponent.java

```diff
@@ -108,15 +108,12 @@ private Element doRender(VariableManager nodeVariableManager, String targetObjec
         DiagramRenderingCache cache = this.props.getCache();
         NodePositionProvider nodePositionProvider = this.props.getNodePositionProvider();
         NodeSizeProvider nodeSizeProvider = new NodeSizeProvider();
-
         UUID nodeId = optionalPreviousNode.map(Node::getId).orElseGet(() -> this.computeNodeId(targetObjectId));
         Optional<Label> optionalPreviousLabel = optionalPreviousNode.map(Node::getLabel);
         String type = nodeDescription.getTypeProvider().apply(nodeVariableManager);
         String targetObjectKind = nodeDescription.getTargetObjectKindProvider().apply(nodeVariableManager);
         String targetObjectLabel = nodeDescription.getTargetObjectLabelProvider().apply(nodeVariableManager);
-
         INodeStyle style = nodeDescription.getStyleProvider().apply(nodeVariableManager);
-
         IDiagramElementRequestor diagramElementRequestor = new DiagramElementRequestor();
 
         //@formatter:off
```

Comment:
I was curious why those lines disappeared but now that I tried to restore them, I understand. If we keep them, we have a checkstyle error since the method is too long. The proper fix is not to remove empty lines and make the code less readable but instead to split the method in smaller ones.